api/nodecontrol/auction-request.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "io.nats.nex.v2.auction_request",
+  "title": "AuctionRequest",
+  "type": "object",
+  "properties": {
+    "arch": {
+      "type": "string",
+      "description": "The architecture auction for",
+      "enum": [
+        "amd64",
+        "arm64"
+      ]
+    },
+    "os": {
+      "type": "string",
+      "description": "The operating system to auction for",
```

Comment:
Similar to previous comment, rearrange so we're not ending the description with `for`

Reply:
blame copilot!

Reply:
This is why I don't use it. I spend more time fixing issues caused by copilot than I save by having copilot help ;)